src/components/Wishlist/WishlistProducts.jsx

```diff
@@ -0,0 +1,44 @@
+import {getWishlistDataHandler, presentInArray, removeFromWishlistHandler} from "../../utils";
+import { useWishlist, useCart } from "../../context";
+import {GoToCartBtn, AddToCartBtn} from "../ProductListing/product-card"
+const WishlistProducts = () => {
+    const token = localStorage.getItem("token");
+    const {wishlistState, wishlistDispatch} = useWishlist();
+    const { cartState } = useCart();
+    return (
+        <>
+            <h1 className="h1 text-center m-5 p-5">Wishlist</h1>
+            
+            {wishlistState.wishlistData.length !== 0 ?  
+            <section className="rui-main--heading-container no-border dismiss-cards flex-container my-10 text-center">
```

Comment:
Suggestion - to make your code a little cleaner, you could create a separate wishlistitem component.